desktop/src/main/java/bisq/desktop/main/account/content/fiataccounts/FiatAccountsDataModel.java

```diff
@@ -50,30 +45,23 @@
 
 class FiatAccountsDataModel extends ActivatableDataModel {
 
+    private final PaymentAccountManager paymentAccountManager;
     private final User user;
     private final Preferences preferences;
-    private final OpenOfferManager openOfferManager;
-    private final TradeManager tradeManager;
-    private final AccountAgeWitnessService accountAgeWitnessService;
     final ObservableList<PaymentAccount> paymentAccounts = FXCollections.observableArrayList();
     private final SetChangeListener<PaymentAccount> setChangeListener;
     private final String accountsFileName = "FiatPaymentAccounts";
     private final PersistenceProtoResolver persistenceProtoResolver;
     private final CorruptedDatabaseFilesHandler corruptedDatabaseFilesHandler;
 
     @Inject
-    public FiatAccountsDataModel(User user,
+    public FiatAccountsDataModel(PaymentAccountManager paymentAccountManager, User user,
```

Comment:
Seems like `FiatAccountsDataModel` and `AltCoinAccountsDataModel` are very similar, especially after changes in this pull request. Shouldn't we introduce some common abstraction for them ?

Reply:
Yes, might be a refactor candidate if fuctionality has duplications there.

Reply:
Yes, but let's keep this PR focused and small.